.rhcicd/clowdapp-connector-splunk.yaml

```diff
@@ -81,6 +81,8 @@ objects:
           value: ${NOTIFICATIONS_CONNECTOR_SEDA_CONCURRENT_CONSUMERS}
         - name: NOTIFICATIONS_CONNECTOR_SEDA_QUEUE_SIZE
           value: ${NOTIFICATIONS_CONNECTOR_SEDA_QUEUE_SIZE}
+        - name: NOTIFICATIONS_CONNECTOR_HTTP_DISABLE_FAULTY_ENDPOINTS
```

Comment:
The SNow template will need the same thing.

Reply:
Sure, sorry I forgot it